### DISCUSSION TITLE
Expo Development Builds

### DISCUSSION THREAD

**Discussion by wjatx** [infinitered/ignite#2473] [Q&A] [answered]
When you're using Ignite, do you still use the regular Expo workflow for making development builds?

For instance:
```
eas build --profile development-simulator --platform ios
```

This is how I make development builds for my other Expo projects, but today I started a new Ignite project from the latest version, and had trouble with the app crashing when I would try to run the build on the iOS simulator.

Is there a better approach to this?

I have packages that aren't in Expo and so can't just use Expo Go.

Thank you in advance for your help.

**Comment by frankcalise** [comment, marked as answer]
Hey @wjatx - you can absolutely use the Expo workflow. I do eas builds all the time, even locally!

The one thing you might have to adjust within your project are the native folders. Ignite v8 was condensed for both vanilla and Expo workflows. If you decide to go full Expo, you should tailor it a bit to do so. Here is a guide on how to do that:
https://ignitecookbook.com/docs/recipes/PristineExpoProject

Likely the most trouble you are running into is the native android/ios directories, which were not generated with Expo in this case. So remove them, build your dev client or run via `npx expo run:ios` and you should be good to go!

**Comment by wjatx** [comment]
That worked - thank you!
